examples/discord_bot.py

```diff
@@ -31,15 +31,15 @@
 @coc.ClanEvents.member_join(tags=clan_tags)
 async def on_clan_member_join(member, clan):
     await bot.get_channel(INFO_CHANNEL_ID).send(
-        "{0.name} ({0.tag}) just " "joined our clan {1.name} ({1.tag})!".format(member, clan)
+        f"{member.name} ({member.tag}) just " "joined our clan {clan.name} ({clan.tag})!"
```

Comment:
There's 2 double quotes in the middle of the string here which makes the second half of the f-string not be evaluated properly